slick.grid.js

```diff
@@ -385,7 +385,13 @@
 
       updateColumnCaches();
       createCssRules();
-      updatePinnedState();
+      if (options.pinnedColumn) {
+        updatePinnedState();
+      } else {
+        topViewport.el.eq(1).hide();
+        contentViewportWrap.el.eq(1).hide();

```

Comment:
I could, but this would turn into a much bigger PR since we're using ints everywhere else in the code.

Mind if we keep it this way for consistency for now?